base/src/main/java/uk/ac/ebi/atlas/experimentimport/analytics/baseline/ProteomicsBaselineAnalyticsInputStream.java

```diff
@@ -125,9 +125,7 @@ public BaselineAnalytics readNext() {
 
             if (!"NA".equalsIgnoreCase(expressionLevelString)) {
                 Double expressionLevel = Double.parseDouble(expressionLevels[i]);
-                if (expressionLevel != 0.0) {
-                    builder.add(new BaselineAnalytics(geneId, StringUtils.removeEnd(assayGroupId, SAMPLE_ABUNDANCE_QUALIFIER), expressionLevel));
-                }
+                builder.add(new BaselineAnalytics(geneId, StringUtils.removeEnd(assayGroupId, SAMPLE_ABUNDANCE_QUALIFIER), expressionLevel));
```

Comment:
I think you really want to keep this condition, and not store the zeros in Solr.

Reply:
Will do, as it’s beyond the scope of this story.

I was envisioning a feature where you could search for e.g. a condition and Expression Atlas could tell you “it’s in this experiment but no baseline expression / differential expression was reported”, rather than “nothing could be found”. We can explore this later.